{"version":1,"resource":"vscode-remote://wsl%2Bubuntu/home/kraoshin/holbertonschool-higher_level_programming/python-exceptions/4-main.py","entries":[{"id":"tqB1.py","timestamp":1737543907621}]}